modules/ROOT/pages/bat-install-task.adoc

```diff
@@ -5,10 +5,28 @@ endif::[]
 
 The Blackbox Automated Testing (BAT) CLI wrapper is a command line wrapper that you install to execute API tests.
 
-== Prerequisite
+== Prerequisite for Linux and MacOS Systems
 
 Before installing the BAT CLI, ensure that AdoptOpenJDK's OpenJDK 8 is installed on your system.
 
+== Prerequisites for Windows 10 Systems
+
+. Install Windows Subsystem Linux (WSL)
++
+The instructions are at https://docs.microsoft.com/en-us/windows/wsl/install-win10.
+. Install Java 8, cURL, and unzip in the Linux distribution that you installed in your WSL.
+. Add the `bat` executable to your `PATH` environment variable.
+.. Open the `~/.bashrc` file (if you are using `bash`) or the `~/.zshrc` file (if you are using `zsh`) for editing. For example, to use the `vi` editor, enter this command:
```

Comment:
```suggestion
.. Open the `~/.bashrc` file (if you are using `bash`) or the `~/.zshrc` file (if you are using `zsh`) for editing. +
For example, to use the `vi` editor, enter the appropriate version of these two commands:
* `vi ~/.bashrc`
* `vi ~/.zshrc`
```
I suggest this because we do not want to join two syntax examples with one narrative word, such as "or" here, or a fragment.